index.html

```diff
@@ -70,25 +70,28 @@
 The <dfn>Server-Timing header field</dfn> is used to communicate one or more metrics and descriptions for the given request-response cycle. The ABNF (Augmented Backus-Naur Form) [[!RFC5234]] syntax for the <a>Server-Timing header field</a> is as follows:
 
 ```ABNF
-  Server-Timing        = #server-timing-metric
-  server-timing-metric = metric  [ OWS ";" OWS description ]
-  metric               = metric-name [ OWS "=" OWS metric-value ]
-  metric-name          = token
-  metric-value         = *DIGIT [ "." 1*DIGIT ]
-  description          = token | quoted-string
+  Server-Timing             = #server-timing-metric
+  server-timing-metric      = metric-name *( OWS ";" OWS server-timing-param )
+  metric-name               = token
+  server-timing-param       = server-timing-param-name OWS [ "=" OWS server-timing-param-value ]
+  server-timing-param-name  = token
+  server-timing-param-value = token / quoted-string
 ```
 
-* Each metric MUST have a name.
-  * Metric names MAY be repeated.
-  * Metric names are defined by the server.
-* Each metric MAY have an optional numeric value.
-* Each metric MAY have an optional description.
-* The user agent MAY surface provided metrics in any order - i.e. the order of metrics in the HTTP header field is not significant.
+See [[!RFC7230]] for definitions of `#`, `*`, `OWS`, `token`, and `quoted-string`.
 
-See [[!RFC7230]] for definitions of `#`, `OWS`, `token`, `DIGIT`, and `quoted-string`.
+Metric names MAY be repeated per response.
+
+The user agent MAY surface provided metrics in any order - i.e. the order of metrics in the HTTP header field is not significant.
+
+This header field is defined with an extensible syntax to allow for future parameters. A user agent that does not recognize particular server-timing-param-name in the Server-Timing header field of a response MUST ignore those tokens and continue processing instead of signaling an error.
+
+To avoid any possible ambiguity, individual server-timing-param-name SHOULD NOT appear multiple times within a server-timing-metric. If any parameter is specified more than once, only the first instance is to be considered. All subsequent occurrences SHOULD be ignored without signaling an error or otherwise altering the processing of the server-timing-metric. This is the only case in which the ordering of parameters within a server-timing-metric is considered to be significant.
```

Comment:
Should we add a note about console warnings of such duplicates, to make debugging easier?

Reply:
... MUST, instead of should and lock down the behavior? We can just say that order of parameters is significant, and duplicate param names are resolved via.. `steps`.